src/test/integration/mocks/server.ts

```diff
@@ -29,15 +29,15 @@ const assertAuth = (credentials: Credentials, authHeader: string | null) => {
 };
 
 export const mockGetOrganization = (
-  organizationId: string,
+  orgs: { id: string; name: string }[],
```

Comment:
then the function name is incorrect ( should be mock get organizations )

Reply:
ad0a7b3a5cf77805c59196a84ca7f3d2e782bc00